slack-plugin/src/main/kotlin/slack/gradle/configureBuildTags.kt

```diff
@@ -49,6 +49,14 @@ internal fun Project.configureBuildScanMetadata(scanApi: ScanApi) {
   scanApi.addGitMetadata(project)
   scanApi.addTestParallelization(project)
   scanApi.addTestSystemProperties(project)
+
+  // Add the gradle enterprise plugin version if present
+  ScanApi::class
+    .java
+    .classLoader
+    .getResource("com.gradle.scan.plugin.internal.meta.buildAgentVersion.txt")
+    ?.readText()
+    ?.let { buildAgentVersion -> scanApi.value("GE Gradle plugin version", buildAgentVersion) }
```

Comment:
nit: can we pull this out to an `addGradleEnterpriseVersion` extension like the ones below?